workspaces/liferay-sample-workspace/client-extensions/liferay-sample-fds-cell-renderer/package.json

```diff
@@ -1,16 +1,26 @@
 {
 	"dependencies": {
-		"@liferay/dxp-7.4": "*",
-		"@liferay/js-api": "0.2.0-pre.0"
+		"@liferay/js-api": "^0.2.0-pre.0",
```

Comment:
We haven't yet standardized how to release `@liferay/js-api` packages, but I'm now thinking that, since we only have one CX type (FDS cell renderer) and we don't expect to have a big explosion in terms of CX type count in the next months, we can maybe make a proper release and decide on how to maintain `@liferay/js-api` as we see samples evolve.

Or, alternatively, we may keep using pre-releases in the beginning and make the first 1.0.0 release when everything has settled enough.

Thoughts? 

I would say we can go with the second option until we have more info...

Reply:
not going to work -> not doing anything different than `0.2.0-pre.0`